main/src/library/Bool.flix

```diff
@@ -59,4 +59,9 @@ mod Bool {
     ///
     pub def <==>(x: Bool, y: Bool): Bool = x == y
 
+    ///
+    /// Guards the side effect `f` if `cond` is true.
```

Comment:
This language seems a bit non-standard to me. I'd just say ```Calls `f` if `cond` is true.``` or refer to `ef` as the side effect, not `f`